src/main/kotlin/net/irisshaders/lilybot/extensions/util/ThreadControl.kt

```diff
@@ -50,16 +48,18 @@ class ThreadControl : Extension() {
 					val member = user.asMember(guild!!.id)
 					val roles = member.roles.toList().map { it.id }
 
-					val moderators = DatabaseHelper.selectInConfig(guild!!.id, DatabaseManager.Config.moderatorsPing)
-					if (moderators.equals("NoSuchElementException")) {
+					// Try to get the moderator ping role from the config.
+					// If a config is not set, inform the user and return@action
+					val moderatorRoleId = DatabaseHelper.selectInConfig(guild!!.id,
+						"moderatorsPing")
+					if (moderatorRoleId == null) {
 						respond {
-							content =
-								"**Error:** Unable to access config for this guild! Please inform a member of staff!"
+							content = "**Error:** Unable to access config for this guild! Is your configuration set?"
```

Comment:
"Unable to access" could mislead. The user could think it's a permission error if the config is missing and vice versa.